sklearn/impute/tests/test_impute.py

```diff
@@ -971,6 +971,58 @@ def test_iterative_imputer_catch_warning():
     assert not np.any(np.isnan(X_fill))
 
 
+@pytest.mark.parametrize("min_value, max_value, correct_output",
+                         [(0,  # scalars cast to the correct arrays?
```

Comment:
instead of putting a comment here we can pass a parameter `ids` to name the different try. It will be easier when running the tests:

```python
@pytest.mark.parametrize(
    "min_value, max_value, correct_output",
    ...,
    ids=["scalars", "list", "None-default", "inf", ...]
)
```

Reply:
Done!